sparse/_coo/common.py

```diff
@@ -1122,7 +1214,7 @@ def _arg_minmax_common(
     from .core import COO
 
     if not isinstance(x, COO):
```

Comment:
An additional check here for `SparseArray` and `spmatrix`, otherwise fail, would be nice too.

Reply:
Sure, added `spmatrix` and `SparseArray` input checks.